docs/user_guide/libraries.rst

```diff
@@ -13,8 +13,8 @@ Here's an example of setting up and importing a macro block as a :class:`.Librar
   chip.add('input', 'verilog', 'mydesign.v')
 
   lib = siliconcompiler.Library(chip, 'mymacro')
-  lib.add('output', stackup, 'lef', 'mylibrary.lef')
-  lib.add('output', stackup, 'gds', 'mylibrary.gds')
+  lib.add('output', stackup, 'lef', 'mymacro.lef')
+  lib.add('output', stackup, 'gds', 'mymacro.gds')
   # ... add more library sources
 
   chip.use(lib)
```

Comment:
on line 21, the chip.add('asic', 'macrolib'), should be updated to mymacro as well